Data/Primitive/ByteArray.hs

```diff
@@ -696,7 +697,9 @@ calcLength :: [ByteArray] -> Int -> Int
 calcLength [] !n = n
 calcLength (x : xs) !n = calcLength xs (sizeofByteArray x + n)
 
+-- | The empty 'ByteArray'.
 emptyByteArray :: ByteArray
+{-# NOINLINE emptyByteArray #-}
```

Comment:
I added a `NOINLINE` pragma for `emptyByteArray`, since all the other empty arrays also have that (I assume this was just an oversight).

Reply:
Yes, just an oversight. We don't really want to allocate extra empty arrays for no reason.